txtorcon/endpoints.py

```diff
@@ -404,7 +404,8 @@ def info_event(msg):
             self.config, self.hidden_service_dir,
```

Comment:
Thanks for pointing that out. You are right, there is no use of creating the HiddenService instance outside the "if" if it is not going to be added anyway to self.config.HiddenServices